python-package/xgboost/__init__.py

```diff
@@ -19,6 +21,12 @@
 except ImportError:
     pass
 
+if sys.version_info[:2] == (3, 5):
+    warnings.warn(
```

Comment:
We can make a schedule.  As 35 is EOL at Sep, we can target 1.2 release assuming 2 months release.

Reply:
Let us open a new thread to discuss when we should drop Python 3.5.